System.IO.Abstractions/IFile.cs

```diff
@@ -15,10 +20,22 @@ public interface IFile
         void AppendAllLines(string path, IEnumerable<string> contents);
         /// <inheritdoc cref="File.AppendAllLines(string,IEnumerable{string},Encoding)"/>
         void AppendAllLines(string path, IEnumerable<string> contents, Encoding encoding);
+#if NETCOREAPP2_0
+        /// <inheritdoc cref="File.AppendAllLinesAsync(string,IEnumerable{string},CancellationToken)"/>
+        Task AppendAllLinesAsync(string path, IEnumerable<string> contents, CancellationToken cancellationToken = default);
```

Comment:
CI is currently failing because the target-typed `default` literal is not available with the compiler we are using. Either change it to `default(CancellationToken)` or upgrade the language version in the csproj.

Reply:
Updated to `default(CancellationToken)` and CI is fixed.